src/claims/claims.controller.ts

```diff
@@ -105,18 +121,58 @@ export class ClaimsController {
   }
 
   @ApiOperation({
-    summary: 'Get CustomClaimType',
-    description:
-      'This endpoint will retrieve the custom claim type by id or name. <br /> ID or name needs to be provided. <br/> If both are provided will fetch using ID',
+    summary: 'Get CustomClaimTypes',
+    description: 'This endpoint fetches CustomClaimTypes that have been registered."',
```

Comment:
```suggestion
    description: 'This endpoint fetches CustomClaimTypes that have been registered"',
```

(nit) I don't think we put in trailing period usually